src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <article id="qa-block" class="article">
+      <div class="orange-box">
+        <p class="paragraph">
+            In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
```

Comment:
80 symbols max per line

Reply:
I changed wrap line in VSCode settings and I saw changes in my index.html, but here, on github, I see that these changes didn't apply. So, can I change this parameter via settings of VS code or I need make lines wrap by hand?

Reply:
if you make some changes, you need to add, commit and deploy again

Reply:
Yes, I inderstand this. I have already made a new commit and a new push. But, although, I don't see changes about line width in updated index.html on github.

Reply:
if you did something like this, that does not work as you expect, you need to edit your code by hand or you can look for some extensions that could do this for you
![image](https://user-images.githubusercontent.com/35385361/111821853-ce041c00-88eb-11eb-9f60-1894a84a58cb.png)